src/UI/Component/Panel/Listing/Listing.php

```diff
@@ -22,4 +22,18 @@ public function getTitle();
 	 * @return \ILIAS\UI\Component\Item\Group[]
 	 */
 	public function getItemGroups();
+
+	/**
+	 * Sets the action drop down to be displayed on the right of the title
+	 * @param \ILIAS\UI\Component\Dropdown\Standard $actions
+	 * @return Listing
+	 */
+	public function withActions(\ILIAS\UI\Component\Dropdown\Standard $actions);
```

Comment:
I would prefer if we would pass a list of shy-buttons (and maybe dividers) and not the dropdown and use the dropdown internally instead. As a consumer, I can only pass a dropdown anyway, so why do I need to create it by myself? Also, as a consumer I'm not allowed to set a label for the dropdown, which the dropdown would allow me to do. Passing shy-buttons only would also allow to use other visualisation options without changes in consumer code.
About the dividers: do we have items, where the menu contains dividers? If not I would just stick to a list of shy-buttons.